src/result_collection.jl

```diff
@@ -90,7 +96,7 @@ function collect_results!(filename, folder;
     end
 
     n = 0 # new entries added
-    existing_files = "path" in names(df) ? df[:,:path] : ()
+    existing_files = "path" in string.(names(df)) ? df[:,:path] : ()
```

Comment:
The behaviour that `names(df)` returns a Vector of strings is only very recent.
I have now encountered this multiple times that DataFrames is restricted to something older
and suddenly DrWatson fails.

Reply:
Yeap the dataframes guys did this breaking change that now `names` return vector of strings. we have to update DrWatson in general to the latest version....

Reply:
yeah, I figured that this wouldn't hurt.
Converting strings to strings shouldn't be too expensive.
And this way we at least know that this will work without having to put a new lower version bound on DataFrames.